frontend/src/hooks/schedule/useScheduleAddModal.ts

```diff
@@ -122,6 +122,11 @@ const useScheduleAddModal = (clickedDate: Date) => {
           showToast('success', '일정이 등록되었습니다.');
           closeModal();
         },
+        onError: (error) => {
+          const response = error as Response;
+          if (response.status === 500)
+            showToast('error', '일정 제목이 최대 글자(200자)를 초과했습니다.');
```

Comment:
PR에 루루가 적어준 정보, 그리고 msw의 로직대로라면 `250` 글자여야 할 것 같은데, 맞을까?
```suggestion
            showToast('error', '일정 제목이 최대 글자(250자)를 초과했습니다.');
```

Reply:
수정할게여